tasks/bootstrap.yml

```diff
@@ -29,6 +29,27 @@
     - inventory_hostname == groups['openio_zk_cluster'][0]
       and openio_zk_status.rc != 0
 
+- name: "Gather PIDs of meta0/meta1 services"
+  shell: "{{ openio_gridinit_cmd }} status2 @meta0 @meta1 | grep {{openio_namespace}} | awk '{print $3;}'"
```

Comment:
Either remove the trailing ";" in this awk command or add it to the next one for consistency's sake. I'd go with the removal here...